chain/src/chain.rs

```diff
@@ -763,13 +763,34 @@ impl ChainService {
                             ) {
                                 Ok((cycles, cache_entries)) => {
                                     let txs_fees =
-                                        cache_entries.into_iter().map(|entry| entry.fee).collect();
+                                        cache_entries.iter().map(|entry| entry.fee).collect();
                                     txn.attach_block(b)?;
                                     attach_block_cell(txn, b)?;
                                     let mut mut_ext = ext.clone();
                                     mut_ext.verified = Some(true);
                                     mut_ext.txs_fees = txs_fees;
                                     txn.insert_block_ext(&b.header().hash(), &mut_ext)?;
+
+                                    // block tx statistics will not include cellbase tx
+                                    // entry.cycles already exclude cellbase
+                                    let tx_stat = BlockTxStat {
+                                        txs_cycles: cache_entries
+                                            .into_iter()
+                                            .map(|entry| entry.cycles)
+                                            .collect(),
+                                        txs_size: resolved
+                                            .iter()
+                                            .filter(|tx| !tx.is_cellbase())
```

Comment:
first tx of block is always cellbase, we may use skip:
```suggestion
                                            .skip(1)
```